backend/mora/service/handlers.py

```diff
@@ -98,42 +104,11 @@ def __init__(self, request: dict, request_type: RequestType):
         else:
             raise NotImplementedError
 
-        if self.request_type == RequestType.EDIT:
-            request = request['data']
-
-        self.prepare_amqp_message(request)
-
-    def prepare_amqp_message(self, request: dict):
-        """Assign and fill ``self.amqp_messages``."""
-        self.amqp_messages = []
-
-        try:  # date = from or to
-            self.date = util.get_valid_from(request)
-        except exceptions.HTTPException:
-            self.date = util.get_valid_to(request)
-        action = {
-            RequestType.CREATE: "create",
-            RequestType.EDIT: "update",
-            RequestType.TERMINATE: "delete",
-        }[self.request_type]
-
-        # both may exist, e.g. for engagement and association
-        if self.employee_uuid:
-            self.amqp_messages.append((
-                'employee',
-                self.role_type,
-                action,
-                self.employee_uuid,
-                self.date,
-            ))
-        if self.org_unit_uuid:
-            self.amqp_messages.append((
-                'org_unit',
-                self.role_type,
-                action,
-                self.org_unit_uuid,
-                self.date,
-            ))
+        if self.do_triggers:
```

Comment:
Instead of using `do_triggers` and checking the same args flag multiple times, do something similar to `force`, check for the flag in the Trigger `run` function instead.